app/lib/encryption/encryption.ts

```diff
@@ -271,22 +287,22 @@ class Encryption {
 			const subsEncrypted = await subCollection.query(Q.where('e2e_key_id', Q.notEq(null))).fetch();
 			// We can't do this on database level since lastMessage is not a database object
 			const subsToDecrypt = subsEncrypted.filter(
-				sub =>
+				(sub: ISubscription) =>
 					// Encrypted message
 					sub?.lastMessage?.t === E2E_MESSAGE_TYPE &&
 					// Message pending decrypt
 					sub?.lastMessage?.e2e === E2E_STATUS.PENDING
 			);
 			await Promise.all(
-				subsToDecrypt.map(async sub => {
+				subsToDecrypt.map(async (sub: any) => {
```

Comment:
Can we remove this any ? maybe `TSubscriptionModel` ?

Reply:
Correct